exercises/shared/.docs/debug.md

```diff
@@ -0,0 +1,18 @@
+# Debug
+
+To help with debugging you can use output to stdout using `env.out.print()`.
+
+If your variable has a reference capability that isn't sendable (as required for env.out.print(), remember you can get a sendable version of your string by clone()ing your string.
```

Comment:
This is something that the test runner (see [the docs](https://exercism.org/docs/building/tooling/test-runners)) has to support. It is usually not easy to do, unless the testing framework has explicit support for capturing any console output. As such, it might be a good idea to omit this file for now, until the test runner has been built and we know for sure it'll support this feature.

Reply:
The intent really of this advice is to provide the pony equivalent of "printf debugging" which would be removed after the tests have passed.

I think if I switch from a print statement that prints to stdout to stderr, the intent may be more clear.

Reply:
(also - checking stdout is very much in my future plans - I was looking for a way to achieve this.  Thank you)